src/etl/bgi_etl.py

```diff
@@ -177,333 +183,443 @@ def _load_and_process_data(self):
+                                        ["https://www.yeastgenome.org",
+                                         "locus",
+                                         local_id,
+                                         "disease"])
+
+                                xref_map = ETLHelper.get_xref_dict(local_cross_ref_id,
+                                                                   prefix,
+                                                                   page,
+                                                                   page,
+                                                                   display_name,
+                                                                   cross_ref_complete_url,
+                                                                   global_xref_id + page)
+                                xref_map['dataId'] = primary_id
+                                cross_references.append(xref_map)
 
                         else:
-                            if prefix == 'PANTHER':  # TODO handle in the resourceDescriptor.yaml
-                                crossRefPrimaryId = crossRef.get('id') + '_' + primary_id
-                                crossRefCompleteUrl = ETLHelper.get_no_page_complete_url(localCrossRefId, ETL.xrefUrlMap, prefix, primary_id)
-                                xrefMap = ETLHelper.get_xref_dict(localCrossRefId, prefix, "gene/panther", "gene/panther", displayName, crossRefCompleteUrl, crossRefPrimaryId + "gene/panther")
-                                xrefMap['dataId'] = primary_id
-                                crossReferences.append(xrefMap)
-
-                            elif prefix == 'RGD':  # TODO handle human generic cross reference to RGD in resourceDescr.
-                                crossRefPrimaryId = crossRef.get('id')
-                                crossRefCompleteUrl = "https://rgd.mcw.edu/rgdweb/elasticResults.html?term="+localCrossRefId
-
-                                xrefMap = ETLHelper.get_xref_dict(localCrossRefId, prefix,
-                                                                        "generic_cross_reference",
-                                                                        "generic_cross_reference", displayName,
-                                                                        crossRefCompleteUrl,
-                                                                        crossRefPrimaryId + "generic_cross_reference")
-                                xrefMap['dataId'] = primary_id
-                                crossReferences.append(xrefMap)
+                            # TODO handle in the resourceDescriptor.yaml
+                            if prefix == 'PANTHER':                     
+                                cross_ref_primary_id = cross_ref.get('id') + '_' + primary_id
+                                cross_ref_complete_url = ETLHelper.get_no_page_complete_url(\
```

Comment:
can the first item in the list be in line with the function call, I find this more readable than the \. ...and the rest like this in this file.